build.gradle

```diff
@@ -25,11 +24,13 @@ dependencies {
     implementation "com.pinterest.ktlint:ktlint-ruleset-standard:$ktlintVers"
 
     compileOnly 'org.jetbrains.kotlin:kotlin-gradle-plugin'
-    compileOnly 'com.android.tools.build:gradle:3.4.2'
+    compileOnly 'com.android.tools.build:gradle:3.5.3'
 
-    testImplementation 'junit:junit:4.12'
-    testImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.1.0'
-    testRuntime 'com.android.tools.build:gradle:3.4.2'
+    testImplementation 'junit:junit:4.13'
+    testImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0'
+    testImplementation 'org.jetbrains:annotations:18.0.0'
```

Comment:
I've bumped dependencies version and added 2 new dependencies
> org.jetbrains:annotations:18.0.0

For better IDE support (mainly the `@Language` annotation)

> org.jetbrains.kotlin:kotlin-gradle-plugin

As that's what I starter to need for tests I wrote

Reply:
> As that's what I started to need for tests I wrote

I was wrong. All the `testRuntime` wasn't actually needed as all `compileOnly` dependencies are added to test runtime using
```
tasks.withType(PluginUnderTestMetadata).configureEach {
    pluginClasspath.from(configurations.compileOnly)
}
```
few lines below.